conversational-skills/procode-skill-sdk-js/src/mock-server/readme.md

```diff
@@ -0,0 +1,59 @@
+# Mock Provider Server
+
+## Goal
+This server is a simple [Express](https://expressjs.com/) server which simulates an example provider. 
+
+## How to use it
+
+### Host it!
+For testing purposes, you can either:
+A. use one of the technologies that expose your localhost to the public domain via tunneling. 
+B. Use IBM CodeEngine
+
+#### IBM Cloud Code Engine Deployment
```

Comment:
Do we want to make this the same level as `Host it` and `Register your Provider`?

Reply:
resolved thru dm, i misread the intent here